apps/codeforafrica/src/lib/data/common/index.js

```diff
@@ -49,6 +49,11 @@ export async function getPageProps(api, context) {
   const {
     docs: [page],
   } = await api.findPage(slug);
+  if (!page && (slug === "404" || slug === "500")) {
+    return {
+      title: "Page not found",
+    };
+  }
```

Comment:
1. Don't different error pages have different titles?
2. Don't they also have subtitles/descriptions?

---

PS: Doesn't 500 page have to be static?

Reply:
@kilemensi The error pages all have the same title and description.

Reply:
Can't be @kelvinkipruto: https://github.com/CodeForAfrica/ui/blob/main/apps/codeforafrica/content/pages/error.md